src/components/Dashboard/Github/dashboard.tsx

```diff
@@ -345,39 +379,72 @@ const DashboardGithub: React.FunctionComponent = () => {
+                                </DropdownItem>
+                              </>
                             )}
-                            {pr.state === 'closed' && (
-                              <DropdownItem key="edit-contribution" isDisabled>
-                                Edit contribution
-                              </DropdownItem>
+                            {draftContributions.find((draft) => draft.branchName == pr.head.ref) && (
+                              <>
+                                <DropdownItem key="view-pr" to={pr.html_url} target="_blank" rel="noopener noreferrer">
+                                  View PR
+                                </DropdownItem>
+                                {pr.state === 'open' && (
+                                  <DropdownItem key="edit-contribution" onClick={() => handleEditClick(pr)}>
+                                    Edit contribution
+                                  </DropdownItem>
+                                )}
+                                {pr.state === 'closed' && (
+                                  <DropdownItem key="edit-contribution" isDisabled>
+                                    Edit contribution
+                                  </DropdownItem>
+                                )}
+                                <DropdownItem key="download-taxonomy" isDisabled>
+                                  Download taxonomy
+                                </DropdownItem>
+                                <Divider component="li" />
+                                <DropdownItem key="delete-draft" isDanger onClick={() => setDeleteDraftContribution(pr.head.ref)}>
+                                  Delete draft
+                                </DropdownItem>
+                              </>
                             )}
                           </DropdownList>
-                          <DropdownItem
-                            key="download-taxonomy"
-                            onClick={() => {
-                              setIsDownloadDone(false);
-                              handleTaxonomyDownload({ branchName: pr.head.ref, isGithubMode: true, setIsDownloadDone });
-                            }}
-                          >
-                            Download taxonomy
-                          </DropdownItem>
                         </Dropdown>
                       )
                     }}
                   >
                     <CardTitle>
                       <Flex alignItems={{ default: 'alignItemsCenter' }} gap={{ default: 'gapSm' }}>
                         {draftContributions.find((draft) => draft.branchName == pr.head.ref) && (
-                          <Label icon={<PficonTemplateIcon />} color="green">
-                            Draft
-                          </Label>
+                          <Flex alignItems={{ default: 'alignItemsCenter' }} gap={{ default: 'gapSm' }}>
```

Comment:
I don't think this `<Flex>` is necessary since these items are already in a `<Flex>`